src/3.1/entrypoint.sh

```diff
@@ -15,40 +15,39 @@ export LD_LIBRARY_PATH=${MUSIC_PATH}/lib:$LD_LIBRARY_PATH
 export PATH=${MUSIC_PATH}/bin:$PATH
 export CPATH=${MUSIC_PATH}/include:$CPATH
 export PYTHONPATH=${MUSIC_PATH}/lib/python3.8/site-packages:$PYTHONPATH
-export NEST_SERVER_RESTRICTION_OFF=true
-export NEST_SERVER_MODULES=nest,numpy
 
-if [[ ! -d /opt/data ]]; then
-    mkdir /opt/data
-fi
+# Create folder if not existed and then change the directory.
+mkdir_cd () {
+    if [[ ! -d $1 ]]; then
+        mkdir $1
+    fi
+    cd $1
+}
 
-if [[ "$1" = 'notebook' ]]; then
-    cd /opt/data
-    exec jupyter-notebook --ip="${IP_ADDRESS}" --port=8080 --no-browser --allow-root
-fi
+MODE="${mode:-$1}"
```

Comment:
Why did you choose a non-standard lower-case variable name over the one in our discussion? Wouldn't a more "scoped" name also help to reduce misunderstandings when many variables are being passed in?

Reply:
Indeed, I changed it to `NEST_CONTAINER_MODE`.